recipe/meta.yaml

```diff
@@ -12,29 +12,29 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
-    - toolchain
     - python
-    - cython
     - setuptools >18.0
     - setuptools_scm >1.5.4
   run:
     - python
+    - asciitree
     - numpy >=1.7
     - fasteners
+    - numcodecs >=0.5.3
```

Comment:
Should be 0.5.4, right?

Reply:
NVM guess we still have it as 0.5.3 in Zarr's `setup.py` and PyPI. Sorry for missing that.

Reply:
No problem, we require >=0.5.3 because that's when the nosetest dependency was removed. Both 0.5.3 and 0.5.4 should work.